project/make/.integration-daemon-start

```diff
@@ -16,6 +16,7 @@ export DOCKER_GRAPHDRIVER=${DOCKER_GRAPHDRIVER:-vfs}
 export DOCKER_EXECDRIVER=${DOCKER_EXECDRIVER:-native}
 
 if [ -z "$DOCKER_TEST_HOST" ]; then
+	unset DOCKER_HOST
```

Comment:
So, this idea needs:
- https://github.com/docker/docker/pull/10747
- https://github.com/docker/docker/pull/10796
- https://github.com/docker/docker/pull/10800

And then it's viable. :+1: :heart_eyes: